### PR TITLE
Updated Documentation Update User

### PR DESCRIPTION
**Pull request by Rjgoolsby**
Updated Documentation Update User to reflect 2 arguments. See https://github.com/kreait/firebase-php/blob/88fdf37531893b775272051970d984b1ba8356eb/src/Firebase/Auth.php#L131

### CODE REVIEW

File: docs/user-management.rst [hunk 1]
```diff
@@ -164,16 +164,17 @@ Updating a user works exactly as creating a new user, except that the ``uid`` pr
 
 .. code-block:: php
 
+    $uid = 'some-uid';
     $properties = [
         'displayName' => 'New display name'
     ];
 
-    $updatedUser = $auth->updateUser($properties);
+    $updatedUser = $auth->updateUser($uid,$properties);
```
Comment (jeromegamez) [inline docs/user-management.rst]: Please add a space after the comma (the small things 😅)

File: docs/user-management.rst [hunk 2]
```diff
@@ -164,16 +164,17 @@ Updating a user works exactly as creating a new user, except that the ``uid`` pr
...
 
     $request = \Kreait\Auth\Request\UpdateUser::new()
         ->withDisplayName('New display name');
 
-    $updatedUser = $auth->updateUser($request);
+    $updatedUser = $auth->updateUser($uid,$request);
```
Comment (jeromegamez) [inline docs/user-management.rst]: Please add a space after the comma (the small things 😅)

File: docs/user-management.rst (hunk 1 above)
Comment (Rjgoolsby) [reply docs/user-management.rst]: :-). Doing now.

File: docs/user-management.rst (hunk 2 above)
Comment (Rjgoolsby) [reply docs/user-management.rst]: :-). Doing now.

**Comment by jeromegamez** [comment]
Thanks @Rjgoolsby! 🌺

**Pull request merged**